contrib/test/integration/e2e-base.yml

```diff
@@ -61,3 +61,7 @@
 
 - name: Enable iptables NAT for the bridge
   command: sysctl -w net.bridge.bridge-nf-call-iptables=1
+
+- name: Relabel /var to be used by kubernetes
+  command: 'semanage fcontext -a -t container_file_t /var'
```

Comment:
restorecon could be another option. cc: @rhatdan

Reply:
whoops I would have needed restorecon when using semanage. I am actually going to try chcon because I don't think these changes need to persist